examples/reference/indicators/LoadingSpinner.ipynb

```diff
@@ -17,9 +17,16 @@
    "cell_type": "markdown",
```

Comment:
I do not know why the label is placed like it is below:

![image](https://github.com/user-attachments/assets/60c790de-0762-422f-9173-3cd2f2e0a1dd)


```python
import panel as pn
import panel_material_ui as pmui

pn.extension()

pmui.Grid(
    "## Monthly Sales",
    "## New Customers",
    "## Revenue Target", 
    pmui.LoadingSpinner(value=42, variant='determinate', bgcolor='light', size=60, with_label=True),
    pmui.LoadingSpinner(value=42, variant="determinate", with_label=True, size=60),
    container=True, columns=3, direction="row").servable()
```